src/chef-server-ctl/bin/chef-server-ctl

```diff
@@ -233,6 +234,9 @@ EOM
 
     # Override reconfigure to skip license checking
     def reconfigure(*args)
+
+      LicenseAcceptance::Acceptor.check_and_persist!("infra-server", ChefServerCtl::VERSION.to_s)
```

Comment:
Do we have access to the config from the user specified `omnibus.rb` here? If so I think we should provide that to `check_and_persist!` [like we do in chef-client](https://github.com/chef/chef/blob/d810b363da8ccdbed92ce92419d6c6e7f2f0835e/lib/chef/application.rb#L258). This gives users another way to accept the license - by adding `chef_license "accept"` into their `omnibus.rb`.

Reply:
We don't have an omnibus.rb visible; but maybe we should look at adding it to the /etc/opscode/chef-server.rb config file.

Reply:
Unfortunately, we're in a bit of a chicken and egg situation with the chef-server.rb config; the parsing of that file happens inside the private-chef cookbook, so it's not accessible here until after the first successful reconfigure.

Reply:
I think `chef-server.rb` was what I was thinking of. If it is not available here that is fine by me. Potentially a future improvement we could make, although it sounds like it would take a lot of refactoring.